assets/js/common.js

```diff
@@ -264,6 +264,13 @@
 							},
 						};
 
+					if (container.data('icon')) {
```

Comment:
Please follow the task description if it describes near-exact implementation, we need support for data-options instead of targeting a single data-icon attribute, then we'll be able to add future changes without extra code. For example, sometimes people are asking how to change some defaults and while it's possible via PHP action and filters hook, it's not possible in JS, so adding support for passing a settings JSON array via data-options is needed